test-specifications/crypto-service/accountUpdateTransaction.md

```diff
@@ -19,111 +17,440 @@ https://github.com/hashgraph/hedera-protobufs/blob/main/services/crypto_update.p
+|---------------------------|--------------------------------------------------|-------------------|--------------------------------------------------------------------------------------------------------------------------------------------|
+| accountId                 | string                                           | optional          | The ID of the account to update.                                                                                                           |
+| key                       | string                                           | optional          | DER-encoded hex string representation for private or public keys. KeyLists and ThresholdKeys are the hex of the serialized protobuf bytes. |
+| autoRenewPeriod           | int64                                            | optional          | Units of seconds                                                                                                                           |
+| expirationTime            | int64                                            | optional          | Epoch time                                                                                                                                 |
+| receiverSignatureRequired | bool                                             | optional          |                                                                                                                                            |
+| memo                      | string                                           | optional          |                                                                                                                                            |
+| maxAutoTokenAssociations  | int32                                            | optional          |                                                                                                                                            |
+| stakedAccountId           | string                                           | optional          |                                                                                                                                            |
+| stakedNodeId              | int64                                            | optional          |                                                                                                                                            |
+| declineStakingReward      | bool                                             | optional          |                                                                                                                                            |
+| commonTransactionParams   | [json object](../commonTransactionParameters.md) | optional          |                                                                                                                                            |
```

Comment:
Link to a json object is not valid.

Reply:
Yeah this links to a markdown that's not merged yet (#208). You approved it but I requested a re-review because I added a lot of changes after you approved. Once that's merged this should be fine.

Reply:
Made a note in the PR description about this.

Reply:
Oh, got it, okay